### DISCUSSION TITLE
QBKG25LM (D1 3gang) configure problem

### DISCUSSION THREAD

**Discussion by z-x-f** [Koenkk/zigbee2mqtt#8179] [Devices]
<pre>
            await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
            await reporting.bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
            await reporting.onOff(device.getEndpoint(1));
            await reporting.onOff(device.getEndpoint(2));
            await reporting.onOff(device.getEndpoint(3));
</pre>
What are the actual uses of these configurations?

**Comment by Koenkk** [comment]
This configuration typically makes the device report state (on/off) changes when controlled physically with the switch.

**Reply by z-x-f** [reply]
But I tried to delete this code, then restart z2m, re-pair the switch, the state of the switch can also be reported normally.
Now this code causes 3 errors to be reported after z2m starts. So if I delete this code, the switch state can also be reported normally, is this code unnecessary?

**Reply by Koenkk** [reply]
> is this code unnecessary?

yes then it is unnecessary indeed, please make a pr to remove it.
